server/templates/pages/src/index.html

```diff
@@ -9,6 +9,7 @@
         <meta name="viewport" content="width=device-width">
         <meta name="referrer" content="origin">
         <meta name="robots" content="noindex,nofollow">
+        <meta name="fxa-content-server/config" content="{{ config }}" />
```

Comment:
Ah yes... I have to tunnel that variable through the build process. Great catch.